api/index.md

```diff
@@ -9,36 +9,6 @@ ms.topic: landing-page
 ---
 
 
-# Q# standard libraries #
+# Q# API reference #
 
-Q# is supported by a range of different useful operations, functions, and user-defined types that comprise the Q# *standard library*.
-The Q# standard library is split into two main parts:
-
-- **The prelude**: operations and functions defined as a part of the target machine and compiler, typically in classical native .NET code.
-  In general, different target machines may have different implementations of the prelude appropriate to each system.
-- **The canon**: operations and functions defined in Q# building on the logic defined in the prelude.
-  The canon implementation is agnostic with respect to target machines.
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
-&nbsp;
\ No newline at end of file
+In addition to the operations and functions of the Q# *standard library*, there are also .NET and Python libraries available for Q#.
```

Comment:
The API reference also covers the three Q# libraries built on top of the standard library: chemistry, numerics, and machine learning.